tr.json

```diff
@@ -4,7 +4,7 @@
     "live" : true,
     "translations" : {
         "meta" : {
-            "thForDate" : false,
+            "thForDate" : true,
```

Comment:
Just to be sure: This change causes "st", "nd" and "th" to be appended to the date. Does this really apply to Turkish?

Reply:
No, soory its fixed.